src/Dfc.CourseDirectory.WebV2/Features/Providers/EditProviderType.cshtml

```diff
@@ -14,6 +14,7 @@
         <h1 class="govuk-heading-l">Provider type</h1>
 
         <form asp-action="Post" asp-all-route-data="@ProviderContext.RouteValues">
+            @Html.HiddenFor(m => m.ProviderId)
```

Comment:
This is not required given `@ProviderContext.RouteValues` above

Reply:
This is a deliberate change to prevent the possibility of accidently updating a different provider's provider type in the case where the provider context is coming from session and the provider context is changed to a different provider in a different window.

Reply:
For admin users the `providerId` should always be in the query string so that scenario cannot happen. It's also inconsistent with how this is handled everywhere else.